1_processing/composition_of_compendia.ipynb

```diff
@@ -806,21 +806,1095 @@
   {
```

Comment:
<p>and same comment with color order here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/core-accessory-interactome/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>